### ISSUE TITLE
Known 'textfield' property in -webkit-appearance attribute is reported as unknown 

### ISSUE THREAD

**Issue by bancer** [CSSLint/csslint#713]
CSS:
```
input[type=number] {
	-moz-appearance: textfield;
	-webkit-appearance: textfield;
}
```
Error:

> Error - Expected (none | button | button-bevel | caps-lock-indicator | caret | checkbox | default-button | listbox	| listitem | media-fullscreen-button | media-mute-button | media-play-button | media-seek-back-button	| media-seek-forward-button	| media-slider | media-sliderthumb | menulist	| menulist-button	| menulist-text	| menulist-textfield | push-button	| radio	| searchfield	| searchfield-cancel-button	| searchfield-decoration | searchfield-results-button | searchfield-results-decoration | slider-horizontal | slider-vertical | sliderthumb-horizontal | sliderthumb-vertical	| square-button	| textarea	| textfield	| scrollbarbutton-down | scrollbarbutton-left | scrollbarbutton-right | scrollbarbutton-up | scrollbargripper-horizontal | scrollbargripper-vertical | scrollbarthumb-horizontal | scrollbarthumb-vertical | scrollbartrack-horizontal | scrollbartrack-vertical) but found 'textfield'. (known-properties)

I found the similar issue https://github.com/CSSLint/csslint/issues/693 but 
1) changing tabs to spaces is against the company's policy, 
2) the same error is reported even when spaces are used instead of tabs.

CSSLint version 1.0.4.

**Comment by toptalo** [comment]
Issue #693 about changing tabs to spaces in parser-lib. My PR fixed this issue is already merged to parser-lib. Now we are waiting update parser-lib in this package.

Parser-lib package is hardcoded to body of CSSLint. Thats why you can't get this update by simply update parser-lib package in your project.

**Comment by toptalo** [comment]
UP!
